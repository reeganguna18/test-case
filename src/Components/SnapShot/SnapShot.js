import React from 'react'

function SnapShot() {
  return (
    <div>
      <p>Snap shot testing</p>
      <a href='/'>Back</a>
    </div>
  )
}

export default SnapShot
