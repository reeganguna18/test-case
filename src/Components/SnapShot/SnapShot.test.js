import { render } from "@testing-library/react";
import SnapShot from "./SnapShot";

test("testing snapshot component", () => {
  const { rensSnaps } = render(<SnapShot />);
  expect(rensSnaps).toMatchSnapshot();
});
