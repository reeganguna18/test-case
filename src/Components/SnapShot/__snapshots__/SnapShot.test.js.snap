// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing snapshot component 1`] = `undefined`;
